nntrainer/models/neuralnet.h

```diff
@@ -313,7 +313,7 @@ class NeuralNetwork : public ml::train::Model {
    * @param[in] X input tensor
    * @retval shared_ptr<const Tensor>
    */
-  sharedConstTensors inference(sharedConstTensors X, bool free_mem = true);
+  sharedConstTensors inference(sharedConstTensors X, bool free_mem = false);
```

Comment:
LGTM!!
how about add commnet about function for "user can find default parms"

Reply:
Please verify backward-compatibility for test-cases and sample applications.
You may start by removing the default value and try to vuild all related test cases and applications. (it will generate build errors in such codes so that you can look at them)

Reply:
@DonghakPark Thanks! I will add a description of "free_mem" parameter.

Reply:
@myungjoo Thank you, I will try to verify backward-compatibility!